### PR TITLE
issue 24

### LINKED ISSUE #24: Footer
- [x] Content (copy/paste)
- [x] Desktop rendering
- [ ] Mobile rendering

Screenshot 2023-09-06 at 10 01 14

### PR DESCRIPTION
**Pull request by paolomoz**
Fix #24 

Test URLs:
- Before: https://main--palladium--hlxsites.hlx.page/es/cares/4-causas/infancia
- After: https://issue-24--palladium--hlxsites.hlx.page/es/cares/4-causas/infancia

### CODE REVIEW

**Comment by paolomoz** [comment]
@Subrat-Code-94 I was playing with footer content structure; this might be easier to work with. Currently only layout done, no styling yet cc @thomsebastin

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,86 @@
...
+}
+
+.footer .center {
+  background-color: var(--highlight-footer-background-color);
+  padding: 54px 15px;
+}
+
+.footer .center input {
+  background: #fff;
+  border: 1px solid #aea780;
```
Comment (thomsebastin) [inline blocks/footer/footer.css:28]: Let's update this with variable name once we make some progress.

**Review by thomsebastin — DISMISSED** [review]

File: blocks/footer/footer.css (hunk 1 above)
Comment (paolomoz) [reply blocks/footer/footer.css:86]: > Let's update this with variable name once we make some progress.

yes

**Comment by paolomoz** [comment]
@Subrat-Code-94 made more progress on the styling based on the new content structure; let me know what you think cc @thomsebastin

**Comment by Subrat-Code-94** [comment]
In Mobile View for slider ,is there any way we can handle that through css ? @paolomoz @thomsebastin

**Comment by paolomoz** [comment]
> In Mobile View for slider ,is there any way we can handle that through css ? @paolomoz @thomsebastin

@Subrat-Code-94 no, you need some javascript, but shouldn't be a lot for what we have here

**Comment by thomsebastin** [comment]
@paolomoz I noticed a few things different from the current footer. Please let me know if these are still WIP.
- Footer currently stretched full width, this is under a `max-width` value in the original.
- Horizontal scroll behaviour is not the same as original.
- Icons do not have `cursor: pointer` applied to them. I understand that they are not linked anywhere now, so maybe this is intentional.
- The disabled button under email input field is not matching with original
- The vertical spacing of items under each accordion is not adequate 
- The horizontal spacing of social icons are different.

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -1,14 +1,301 @@
...
+
+.footer .center ul li ul {
+  margin-top: 13px;
+  font-size: var(--body-font-size-xs);
+}
+
+.footer .top > div:first-of-type {
+  margin-bottom: 5px;
+  text-align: center;
+  border-bottom: 1px solid var(--highlight-footer-background-color);
```
Comment (thomsebastin) [inline blocks/footer/footer.css:179]: I usually don't name variables based on where they are used. If the same color is used elsewhere, it doesn't read well.

**Comment by paolomoz** [comment]
> @paolomoz I noticed a few things different from the current footer. Please let me know if these are still WIP.
> 
> * Footer currently stretched full width, this is under a `max-width` value in the original.
> * Horizontal scroll behaviour is not the same as original.
> * Icons do not have `cursor: pointer` applied to them. I understand that they are not linked anywhere now, so maybe this is intentional.
> * The disabled button under email input field is not matching with original
> * The vertical spacing of items under each accordion is not adequate
> * The horizontal spacing of social icons are different.

Thank you for the very useful feedback

**Comment by paolomoz** [comment]
> * Horizontal scroll behaviour is not the same as original.
> * Icons do not have `cursor: pointer` applied to them. I understand that they are not linked anywhere now, so maybe this is intentional.

These 2 were intentional, I would like to hear feedback from UI owners
I hope I was able to fix the other points properly

**Pull request merged**